src/sync/impl/helpers.js

```diff
@@ -54,10 +55,29 @@ export function prepareCreateFromRaw<T: Model>(collection: Collection<T>, dirtyR
   })
 }
 
-export function prepareUpdateFromRaw<T: Model>(record: T, updatedDirtyRaw: DirtyRaw): T {
+export function prepareUpdateFromRaw<T: Model>(
+  record: T,
+  updatedDirtyRaw: DirtyRaw,
+  log: SyncLog,
+): T {
+  // Note COPY for log - only if needed
+  const logConflict = !!record._raw._changed
+  const logLocal = logConflict ? { ...record._raw } : {}
+  const logRemote = logConflict ? { ...updatedDirtyRaw } : {}
```

Comment:
Shouldn't this be done only if user wants logs?

Reply:
sure, I can do that

Reply:
fixed